Wed 15 Aug 2018 00:35:49 : DEBUG : logging.py : 33 : successfully get data from database
